docs/pull_request.md

```diff
@@ -0,0 +1,208 @@
+# Pull requests
+
+## Introduction
+
+This document gives some clue about how to efficiently manage Pull Requests (PR). This document is a first draft and may be improved later.
+
+## Who should read this document?
+
+Every pull request reviewers, but also probably every ones who submit PRs.
+
+## Submitting PR
+
+### Who can submit pull requests?
+
+Basically every one who wants to contribute to the project! But there are some rules to follow.
+
+#### Humans
+
+People with write access to the project can directly clone the project, push their branches and create PR.
+
+External contributors must first fork the project and create PR to the mainline from there.
+
+##### Draft PR?
+
+Draft PR can be created when the submitter does not expect the PR to be reviewed and merged yet. It can be useful to publicly show the work, or to do a self-review first.
+
+Draft PR can also be created when it depends on other un-merged PR.
+
+In any case, it is better to explicitly declare in the description why the PR is a draft PR.
+
+Also, draft PR should not stay indefinitely in this state. It may be removed if it is the case and the submitter does not update it after a few days.
+
+##### PR Review Assignment
+
+We use automatic assignment for PR reviews. A PR is automatically routed by GitHub to a team member using the round robin algorithm. The process is the following:
+
+- The PR creator assigns the [element-android](https://github.com/orgs/vector-im/teams/element-android) team as a reviewer. They can skip this process and assign directly a specific member if they think they should take a look at it.
```

Comment:
thinking out loud (not a blocker for this PR) 

could we automate this assignment? similar to how we apply the triage project to new issues~
I'm happy to have a go at updating the github actions

Reply:
This is a good idea but we still want to be able to assign someone directly. If we change the selected reviewer afterwards, it will create noise.

There is a missing paragraph from the original doc that was used to create this section. I will add a comment.

Reply:
I wouldn't mind adding specific reviewers as additional reviewers :see_no_evil: I wonder if when a specific person is being added as a reviewer it's a sign that the area could benefit from more eyes for sharing the knowledge